libtailscale/net.go

```diff
@@ -170,6 +170,22 @@ func (b *backend) updateTUN(service IPNService, rcfg *router.Config, dcfg *dns.O
 			return err
 		}
 	}
+
+	for _, route := range rcfg.LocalRoutes {
+		// If route is localhost, do not exclude.
```

Comment:
Why do we not exclude localhost? And what about IPv6's loopback address?

Reply:
VpnService throws an exception if it's a loopback address - https://android.googlesource.com/platform/frameworks/base/+/c741553/core/java/android/net/VpnService.java#303. Updated to check for IPv6!

Reply:
Upated to check IPv6 too! VpnService throws an exception for loopback addresses: https://android.googlesource.com/platform/frameworks/base/+/c741553/core/java/android/net/VpnService.java#303